From 4a24e17e200094718390cf4d5b3376bd0131cbfd Mon Sep 17 00:00:00 2001
From: Chad Dupuis <cdupuis@redhat.com>
Date: Thu, 21 Aug 2014 13:05:27 -0400
Subject: [ethernet] qlcnic: Fix typo in printk

Message-id: <1408626422-3941-44-git-send-email-cdupuis@redhat.com>
Patchwork-id: 88600
O-Subject: [PATCH RHEL 7.1 043/138] treewide: Fix typo in printk
Bugzilla: 1089352
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Masanari Iida <standby24x7@gmail.com>

Bug 1089352 (https://bugzilla.redhat.com/show_bug.cgi?id=1089352)

net commit id 6d3be300c658ce3f1777a6b1279414c62bc7d817

Correct spelling typo within various part of the kernel

Signed-off-by: Masanari Iida <standby24x7@gmail.com>
Acked-by: Randy Dunlap <rdunlap@infradead.org>
Signed-off-by: Jiri Kosina <jkosina@suse.cz>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/qlogic/qlcnic/qlcnic_83xx_hw.c b/drivers/net/ethernet/qlogic/qlcnic/qlcnic_83xx_hw.c
index 0c32e20..388c1d9 100644
--- a/drivers/net/ethernet/qlogic/qlcnic/qlcnic_83xx_hw.c
+++ b/drivers/net/ethernet/qlogic/qlcnic/qlcnic_83xx_hw.c
@@ -1730,7 +1730,7 @@ static void qlcnic_extend_lb_idc_cmpltn_wait(struct qlcnic_adapter *adapter,
  struct qlcnic_hardware_context *ahw = adapter->ahw;
  int temp;
 
- netdev_info(adapter->netdev, "Recieved loopback IDC time extend event for 0x%x seconds\n",
+ netdev_info(adapter->netdev, "Received loopback IDC time extend event for 0x%x seconds\n",
       ahw->extend_lb_time);
  temp = ahw->extend_lb_time * 1000;
  *max_wait_count += temp / QLC_83XX_LB_MSLEEP_COUNT;
-- 
1.7.1